sources/packages/backend/apps/queue-consumers/src/services/student-file/student-file.service.ts

```diff
@@ -49,7 +56,18 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
         UNABLE_TO_SCAN_FILE,
       );
     }
-    await this.updateFileScanStatus(studentFile.uniqueFileName, isInfected);
+    if (isInfected) {
+      processSummary.warn("Virus found.");
+      const fileInfo = path.parse(studentFile.fileName);
+      studentFile.fileName = `${fileInfo.name}${INFECTED_FILENAME_SUFFIX}${fileInfo.ext}`;
+    } else {
+      processSummary.info("No virus found.");
+    }
+    await this.updateFileScanStatus(
+      studentFile.uniqueFileName,
+      isInfected,
+      studentFile.fileName,
```

Comment:
Minor, but using a `studentFile.fileName` as a storage for a transient variable can be misleading.
For this situation I would prefer to create a variable initialized with the `studentFile.fileName` and then changed the infected file name if needed.